docs/en/deployment/shared_data/minio.md

```diff
@@ -87,21 +87,6 @@ Whether to use the [AWS SDK default credentials provider chain](https://docs.aws
 - `true`
 - `false` (Default).
 
```

Comment:
check `aws_s3_use_aws_sdk_default_behavior`  in row 83, this parameter is not mentioned in the Chinese doc